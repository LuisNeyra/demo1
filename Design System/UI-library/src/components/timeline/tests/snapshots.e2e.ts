import { snapIt } from '../../../test';

const component = 'timeline';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('without items', '<blaze-timeline></blaze-timeline>');
    snap(
      'with items',
      `<blaze-timeline>
        <blaze-timeline-item>Item 1</blaze-timeline-item>
        <blaze-timeline-item>Item 2</blaze-timeline-item>
        <blaze-timeline-item>Item 3</blaze-timeline-item>
      </blaze-timeline>`
    );
    snap(
      'all the attributes set',
      `<blaze-timeline alternate loading>
        <blaze-timeline-item left type="brand">Item 1</blaze-timeline-item>
        <blaze-timeline-item last>Item 2</blaze-timeline-item>
        <blaze-timeline-item loading>Item 3</blaze-timeline-item>
      </blaze-timeline>`
    );
  });
});
