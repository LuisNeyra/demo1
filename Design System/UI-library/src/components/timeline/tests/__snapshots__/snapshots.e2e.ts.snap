// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timeline renders all the attributes set 1`] = `"<blaze-timeline alternate=\\"\\" loading=\\"\\" class=\\"hydrated\\"><!----><ol class=\\"o-timeline o-timeline--alternate o-timeline--loading\\"> <blaze-timeline-item left=\\"\\" type=\\"brand\\" class=\\"hydrated\\"><!----><li class=\\"c-timeline-item c-timeline-item--brand c-timeline-item--left\\"><div class=\\"c-timeline-item__body\\">Item 1</div></li></blaze-timeline-item> <blaze-timeline-item last=\\"\\" class=\\"hydrated\\"><!----><li class=\\"c-timeline-item c-timeline-item--last\\"><div class=\\"c-timeline-item__body\\">Item 2</div></li></blaze-timeline-item> <blaze-timeline-item loading=\\"\\" class=\\"hydrated\\"><!----><li class=\\"c-timeline-item c-timeline-item--loading\\"><div class=\\"c-timeline-item__body\\">Item 3</div></li></blaze-timeline-item> </ol></blaze-timeline>"`;

exports[`timeline renders with items 1`] = `"<blaze-timeline class=\\"hydrated\\"><!----><ol class=\\"o-timeline\\"> <blaze-timeline-item class=\\"hydrated\\"><!----><li class=\\"c-timeline-item\\"><div class=\\"c-timeline-item__body\\">Item 1</div></li></blaze-timeline-item> <blaze-timeline-item class=\\"hydrated\\"><!----><li class=\\"c-timeline-item\\"><div class=\\"c-timeline-item__body\\">Item 2</div></li></blaze-timeline-item> <blaze-timeline-item class=\\"hydrated\\"><!----><li class=\\"c-timeline-item\\"><div class=\\"c-timeline-item__body\\">Item 3</div></li></blaze-timeline-item> </ol></blaze-timeline>"`;

exports[`timeline renders without items 1`] = `"<blaze-timeline class=\\"hydrated\\"><!----><ol class=\\"o-timeline\\"></ol></blaze-timeline>"`;
